### PR TITLE
Dev file to see button, webpack aliases, button example

### PR DESCRIPTION
**Pull request by maxrbaldwin**
1. Created an `index.html` file that references the script in the build directory. So that we can see the button in action.
2. Added webpack aliases. See the `.bablerc` for examples.
3. The added `index.html` has a simple button.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-const test2 = {
-  ...test,
-  keyTwo: 2,
-}
+placements.forEach(function placeReportButtons(div, i) {
+  const reportButton = document.createElement('button');
 
-dfpAdapter()
+  reportButton.classList.add('report-ad-button');
+  reportButton.innerHTML = "<span>Report Ad!</span>";
```
Comment (trinhathu) [inline src/index.js]: Single quotes!

File: src/index.js [hunk 2]
```diff
@@ -1,14 +1,18 @@
-import dfpAdapter from './adapters/dfp';
+import handleClick from '@handlers/onclick';
 
-console.log('run');
+const config = window.reportAdsConfig || {};
+const { selector='.ad' } = config;
```
Comment (trinhathu) [inline src/index.js]: Remove the default or put a space around the =?

File: src/index.js [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
-  key: 1,
-}
+const placementsList = document.querySelectorAll(selector);
+const placements = Array.prototype.slice.apply(placementsList);
 
-const test2 = {
-  ...test,
-  keyTwo: 2,
-}
+placements.forEach(function placeReportButtons(div, i) {
```
Comment (trinhathu) [inline src/index.js]: =>

File: src/index.js (hunk 1 above)
Comment (maxrbaldwin) [reply src/index.js]: 👍

File: src/index.js (hunk 2 above)
Comment (maxrbaldwin) [reply src/index.js]: I kept the default. I renamed it. Added the space.

File: src/index.js (hunk 3 above)
Comment (maxrbaldwin) [reply src/index.js]: 👎

Comment (emilymalecbrown) [reply src/index.js]: 👍

**Pull request merged**